sadcowpng/sharedspaces/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+
+# Tuple list used for Multiple Choice Field
+Noise_Level_Choices = (
+    ("1", "None"),
+    ("2", "Faint"),
+    ("3", "Moderate"),
+    ("4", "Loud"),
+    ("5", "Very Loud")
+)
+
+
+# For for creating the specific space and also to modify it.
+class CreateSpaceForm(forms.Form):
+    space_name = forms.CharField(label='Name of the Shared Space:', max_length=500, strip=True)
+    space_description = forms.CharField(label='Description:', max_length=1000, strip=True)
+    space_max_capacity = forms.IntegerField(label='Maximum Occupancy:', min_value=0)
+    space_noise_level_allowed = forms.MultipleChoiceField(label='Noise level allowed:', choices=Noise_Level_Choices,
+                                                          required=True)
+    space_noise_level = forms.MultipleChoiceField(label='Noise level:', choices=Noise_Level_Choices, required=True)
+
+    # Might switch boolean fields to multiple choice for clarity
```

Comment:
This comment mentions 'Might'. I see that we're still using BooleanFields here. Make sure that this final decision is reflected in the comment, as I'm not sure if you switched from something else to boolean fields for clarity or vice versa.

Reply:
removed